Packs/VMware/Integrations/VMware/VMware.yml

```
@@ -208,11 +259,54 @@ script:
     - description: VM UUID of virtual machine to get events of.
       name: vm-uuid
       required: true
-    - defaultValue: VmGuestRebootEvent,VmGuestShutdownEvent,VmPoweredOnEvent,VmPoweredOffEvent,VmSuspendedEvent
+    - name: event-type
       description: Type of events to get, given in CSV (i.e.,  VmGuestRebootEvent,VmGuestShutdownEvent).
-      name: event-type
+      required: false
+      isArray: false
+      defaultValue: VmGuestRebootEvent,VmGuestShutdownEvent,VmPoweredOnEvent,VmPoweredOffEvent,VmSuspendedEvent
+    - name: start-date
+      description: Event's start date.
+      required: false
+      isArray: false
+    - name: end-date
+      description: Event's end date
+      required: false
+      isArray: false
+    - name: user
+      description: User name.
```

```suggestion
      description: If specified, only events triggered by the given user are retrieved.
```
correct me if I'm wrong on this